packages/block-library/src/gallery/edit.js

```diff
@@ -122,6 +122,7 @@ function GalleryEdit( props ) {
 	const imageData = useSelect(
 		( select ) => {
 			if (
+				! innerBlockImages ||
 				innerBlockImages.length === 0 ||
```

Comment:
Would this tweak make it a little more concise but still clear?
```suggestion
				! innerBlockImages?.length ||
```

Reply:
yeh, that is tidier, done.